### PR TITLE
chore(workflow): update add-to-projects.yml

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue longhorn#9411

#### What this PR does / why we need it:

1. Move ticket to "In Progress" after reopening it.
2. Don't move a "Resolved" or "Closed" ticket to "In Progress" if non-longhorn user adds a comment.
3. Move ticket to "Closed" after closing it as not planned.
4. Move ticket to "Resolved" after closing it.

### CODE REVIEW

**Comment by derekbit** [comment]
Ready for review. cc @innobead

**Comment by PhanLe1010** [comment]
> 1. Move ticket to "In Progress" after reopening it.
> 1. Don't move a "Resolved" or "Closed" ticket to "In Progress" if non-longhorn user adds a comment.

I think the bot shouldn't move the ticket back once the ticket has been marked as resolved no matter who adds a new comment or it is reopened. There are cases when we have accepted the issue (which means it should no longer be a community ticket) and the dev is having some more conversations with the community user when doing the development. In these cases, the bot will keep moving the ticket back to `in-progress` which I feel is incorrect.

In short, I think the bot should not do anything once the community coordinator moves the ticket to resolved column. If it is needed to be moved back, it can be moved back manually instead. 

WDYT?

**Comment by PhanLe1010** [comment]
I am looking forward to having this PR merged. Currently, the bot keeps moving the tickets back and fort.

**Review by PhanLe1010 — COMMENTED** [review]
> 6. If a comment is added or edited and the status is not Resolved or Closed by
   longhorn member, update to In Progress
   
Does this one mean a comment from community user on a not Resolved or Closed issue will not trigger the bot to move the ticket back to `in-progress`?

**Comment by PhanLe1010** [comment]
I think what we want is
> If a comment is added or edited by "community"  and the status is not Resolved or Closed member, update to In Progress

But what the current comment is:
> If a comment is added or edited and the status is not Resolved or Closed by "longhorn" member, update to In Progress


WDYT?

**Review by PhanLe1010 — APPROVED** [review]
LGTM

Thank you for the feature!
